DOCUMENTATION.md

```diff
@@ -134,19 +135,34 @@ This type of object is constructed dynamically from the message received by the
 ## ApplicationClient
 The class constructor can be called following this scheme:
 ```python
-ApplicationClient(app_id, access_key_or_token, [net_address], [certificate], [discovery_address])
+ApplicationClient(app_id, access_key_or_token, **kwargs)
 ```
 - `app_id`: **string**  this the name given to the application when it was created.
 ![Screenshot of the console with app section](./images/app-console.png?raw=true)
 - `access_key_or_token`: **string**  this can be found at the bottom of the application page under **ACCESS KEYS**. You will need a key allowing you to change the settings if you wish to update your application.
-- `net_address`: **string**  this  is the address of the handler to which the application was registered. It needs to be provided as a `net_address=value` argument when calling the constructor.
-- `certificate`: **string**  this is the content of the certificate used to connect in a secure way to the handler. It needs to be provided as a `certificate=value` argument when calling the constructor.
-- `discovery_address`: **string**  this is the address of the discovery server to use in order to find back the address of the handler to which the application in registered. It needs to be provided as a `discovery_address=value` argument when calling the constructor.
-- `path_to_key=`: **string**  this is the absolute path to the file which contains the key from which the token you wish to use, is signed. It needs to be provided as a `path_to_key=value` argument when calling the constructor.
+
+The parameters **\*\*kwargs** can be the following:
+- `net_address`: **string**  this  is the address of the handler to which the application was registered. It's composed by the hostname and the port number, example: `handler.eu.thethings.network:1904`.
+- `certificate`: **string**  this is the content of the certificate used to connect in a secure way to the handler. For example:
```

Comment:
I have to pass it as the string to the ApplicationClient because that's how I get it back from the discovery server when opening a handler which gives you the possibility to create an ApplicationClient object. I decided to give the possibility to pass the path in the HandlerClient constructor then.